.gitignore

```diff
@@ -9,3 +9,9 @@ env
 .ipynb_checkpoints
 
 src/_build
+
+.DS_Store
+desktop.ini
+
+.idea/
+venv/
```

Comment:
Realize I should probably grab one of the python .gitignore template from [there](https://github.com/github/gitignore)

Reply:
resolving, as this is tracked in #256 now